### PR TITLE
JCLOUDS-1596

### PR DESCRIPTION
**Pull request by SATYANAN-ANAND**

### CODE REVIEW

File: providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/AzureComputeApi.java [hunk 1]
```diff
@@ -54,244 +55,291 @@
...
-    */
-   @Delegate
-   VirtualNetworkGatewayConnectionApi getVirtualNetworkGatewayConnectionApi(@PathParam("resourcegroup") String resourcegroup);
-   
-   /**
-    * Returns the information about the current service principal.
-    */
-   @Provides
-   Supplier<ServicePrincipal> getServicePrincipal();
+
```
Comment (gaul) [inline providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/AzureComputeApi.java:66]: Please revert whitespace changes.

**Review by gaul — CHANGES_REQUESTED** [review]

File: providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/AzureComputeApi.java (hunk 1 above)
Comment (SATYANAN-ANAND) [reply providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/AzureComputeApi.java]: Hi,
I have reverted the changes.

**Review by gaul — APPROVED** [review]
This looks reasonable to me.  @jmspring any feedback?

**Comment by SATYANAN-ANAND** [comment]
Any feedback/corrections on this implementation. Please let me know.

**Comment by gaul** [comment]
Thank you for your contribution @SATYANAN-ANAND!

**Pull request merged**